subsys/can/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+if(CONFIG_CAN_STACKS)
+  zephyr_interface_library_named(CAN_STACKS)
+
+  zephyr_library()
+  zephyr_library_sources(common.c)
+  zephyr_library_link_libraries(CAN_STACKS)
+
+
+  add_subdirectory_ifdef(CONFIG_CANOPEN    ./canopen)
```

Comment:
Should we have a proper name space for CAN specific Kconfig options. So something like CONFIG_CAN_OPEN, CONFIG_CAN_DEVICENET etc. This way it is much easier to find relevant CAN specific options by grepping.